test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNetCore/AspNetCoreMvcTestBase.cs

```diff
@@ -39,7 +39,7 @@ protected AspNetCoreMvcTestBase(string sampleAppName, ITestOutputHelper output,
             HttpClient.DefaultRequestHeaders.Add(HeaderName1WithMapping, HeaderValue1);
             HttpClient.DefaultRequestHeaders.Add(HeaderName2, HeaderValue2);
             SetEnvironmentVariable(ConfigurationKeys.HeaderTags, $"{HeaderName1UpperWithMapping}:{HeaderTagName1WithMapping},{HeaderName2},{HeaderName3}");
-            SetEnvironmentVariable(ConfigurationKeys.HttpServerErrorStatusCodes, "400-403, 500-501-234, s342, 500");
+            SetEnvironmentVariable(ConfigurationKeys.HttpServerErrorStatusCodes, "400-403, 500-501-234, s342, 500-503");
```

Comment:
What do you think of removing the invalid values here? `500-501-234, s342`
We shouldn't be testing the parsing logic here for this setting. That should be a different unit test.
(feel free to punt this)

Reply:
Yeah, I agree, we shouldn't be testing that here

Reply:
Good point, feel free to remove the middle two groups

Reply:
I remove these invalid values in https://github.com/DataDog/dd-trace-dotnet/pull/1525/commits/7e5f8d3f94180528d461007e093201768485435b

Added them as a test case to `TracerSettingsTests.ParseHttpCodes` instead

Reply:
Thanks!